pyxem/signals/__init__.py

```diff
@@ -93,3 +88,7 @@
     "VirtualDarkFieldImage",
     "InSituDiffraction2D",
 ]
+
+
+def __dir__():
+    return sorted(__all__)
```

Comment:
This is only needed for lazy loading of modules, I think. Do we plan to add that to pyxem?

Reply:
Ahh that's on my list (eventually). I think I must have just copied it without knowing what it was for....

Reply:
We are going to leave that for a separate PR though :)

Reply:
Sounds good! I suggest to remove it here, then.